README.md

```diff
@@ -1,17 +1,20 @@
-# Go Bindings for GLFW 3.1
-
-* **ATTENTION:** As of GLFW 3.1 we break API. See Changelog below.
-* See [here](http://godoc.org/github.com/go-gl/glfw/v3.1/glfw) for documentation.
-* You can help by submitting examples to [go-gl/examples](http://github.com/go-gl/examples).
+# GLFW 3.1 for Go [![GoDoc](https://godoc.org/github.com/go-gl/glfw/v3.1/glfw?status.svg)](https://godoc.org/github.com/go-gl/glfw/v3.1/glfw)
 
 ## Installation
 
-* Installation is easy, just `go get github.com/go-gl/glfw/v3.1/glfw` and be done (*GLFW sources are included so you don't have to build GLFW on your own*)!
-* Go 1.4 is required on Windows (otherwise you must use MinGW v4.8.1 exactly, see [Go issue 8811](https://code.google.com/p/go/issues/detail?id=8811)).
+* GLFW C library source is included and built automatically as part of the Go package. But you need to make sure you have dependencies of GLFW:
+	* On OS X, you need Xcode for required headers and libraries.
+	* On Linux, you need `libgl1-mesa-dev` and `xorg-dev` packages.
```

Comment:
I know, but it's what GLFW suggests.

From http://www.glfw.org/docs/latest/compile.html#compile_deps:

> ### Dependencies for Linux and X11
>
> To compile GLFW for X11, you need to have the X11 and OpenGL header packages installed, as well as the basic development tools like GCC and make. For example, on Ubuntu and other distributions based on Debian GNU/Linux, you need to install the **`xorg-dev`** and `libglu1-mesa-dev` packages.